torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -420,11 +420,16 @@ class ViewAndMutationMeta:
     # Filled after tracing joint function.
     num_backward_tokens: int = 0
 
+    # Number of rng states that will get thread into the forward and backward for
+    # cudagraph compatible run_and_save_rng
+    num_graphsafe_rng_states: int = 0
```

Comment:
one callout is that I think this will be included in the AOT cache key by default, although it's unnecessary (since it can be inferred directly from the ops in the dynamo graph that we compute the cache key from) - probably doesn't matter though, fyi @jamesjwu